### DISCUSSION TITLE
Vercel is not deploying my latest commit

### DISCUSSION THREAD

**Discussion by Veer0x1** [vercel/community#2906] [Help] [answered]
### Summary

I have deployed my next website on vercel, previously it used to automatically deploy my new commit. Today I pushed some changes on my GitHub repo but vercel is not deploying the latest commit. I'm don't know what is the reason behind it. Help me out!!!
<img width="1440" alt="Screenshot 2023-06-18 at 11 59 15 PM" src="https://github.com/vercel/community/assets/96714915/f05299a5-3007-4a7e-b57e-6d59b9d4a6cc">
This are my latest commit...
<img width="1464" alt="Screenshot 2023-06-18 at 11 58 46 PM" src="https://github.com/vercel/community/assets/96714915/ab10f53c-8a0d-4c47-bf5e-f67506f279de">

### Steps to Reproduce

I don't know how to replicate

**Comment by mehditareghi** [comment]
I think your project has at least one build error. Try to build your project with proper command (e.g. for nextjs project, use "npm run build"), it should show you the error...

**Reply by ecelliitbhu** [reply]
I tried to build it using ```yarn build``` locally, it built successfully.
<img width="1440" alt="Screenshot 2023-06-19 at 12 01 02 PM" src="https://github.com/vercel/community/assets/87716183/40ebeb8e-4197-4b92-bcaf-165bbeefa33c">
<img width="1440" alt="Screenshot 2023-06-19 at 12 01 25 PM" src="https://github.com/vercel/community/assets/87716183/dfff15e5-4aa1-450f-953f-daadb1d97055">
This build log from vercel, actually vercel is not running build my latest commit. It look like it doesn't; have context that there is new commit, is there any way I can tell vercel to build by app on particular commit?

**Reply by smaeda-ks** [reply]
@ecelliitbhu what's the full git commit SHA that didn't trigger deployments?

**Reply by ecelliitbhu** [reply]
<img width="1440" alt="Screenshot 2023-06-19 at 1 04 37 PM" src="https://github.com/vercel/community/assets/87716183/0796c553-7e0a-41b8-b7cd-f23ceba27370">
This is the full commit that ```208c95843649a87e7ca358f64ecc941ca21a0df8```

**Reply by smaeda-ks** [reply, marked as answer]
I can't find any logs. Please try reconnecting the GitHub repository with your project. It sounds like it's been disconnected

**Reply by ecelliitbhu** [reply]
How to reconnect git repo? On my vercel dashboard I'm not able to see any such option

**Reply by smaeda-ks** [reply]
go to the project, Settings -> Git
